src/webui/www/public/scripts/contextmenu.js

```diff
@@ -284,6 +286,11 @@ var TorrentsTableContextMenu = new Class({
                 all_are_force_start = false;
             else
                 there_are_force_start = true;
+
+            if (data['auto_tmm'] === true)
+                there_are_auto_tmm = true;
+            else
+                all_are_auto_tmm = false;
```

Comment:
I didn't understand this if-else block...
But I think it's used the same way as other similar things.